.gitignore

```diff
@@ -1 +1,3 @@
-exampleSite/public
+exampleSite/public/
+exampleSite/resources/
```

Comment:
Bep suggested to commit these files into. We can discuss this, but this is out of scope if this PR for me.